governance-and-automation/automated-workflows/policy-automation.md

```diff
@@ -0,0 +1,84 @@
+---
+description: This article explains how to set up and utilize the policy automation feature. 
+
+---
+
+# Policy Automation
+
+The Policy Automation feature **lets you create rules for auto-applying policies based on specific metadata and built-in properties**. 
```

Comment:
... based on:
ovdje bih naveo točno sve po čemu se može applyati rule/policy
Name, Privacy, Sensitivity Label, Sharing Settings, URL, Workspace Type

Reply:
+ dodao bih kao u RN, 'with custom metadata to be added soon!' ili tako neki teaser